src/framework/components/anim/component-layer.js

```diff
@@ -87,6 +88,27 @@ class AnimComponentLayer {
         this._mask = mask;
```

Comment:
you should add Debug.deprecated message to this now private function, suggesting people to use the .mask instead.

Reply:
Will do